### PR TITLE
Updating Page: How-Does-Orchestration-Work

### PR DESCRIPTION
**Pull request by amessbee**
Mainly expanding ICA description based on Dan's comments.

### CODE REVIEW

**Review by Jovonni — APPROVED** [review]

**Comment by amessbee** [comment]
Ref: @Jovonni's comment from another PR that is relevant here
https://github.com/Agoric/documentation/pull/1201#discussion_r1775500217

File: main/guides/orchestration/how-orch-works.md [hunk 1]
```diff
@@ -1,37 +1,115 @@
 # How Orchestration Works
-Blockchain orchestration fundamentally relies on protocols and mechanisms that enable blockchains to communicate and transact with each other securely and efficiently. The primary goals of blockchain orchestration are:
 
-- **Interoperability:** Allowing different blockchain networks to interact and transact with each other.
-- **Scalability:** Enabling multiple blockchains to work together without compromising performance.
-- **Security:** Ensuring that cross-chain transactions are secure and trustworthy.
+Blockchain orchestration relies on protocols and mechanisms that allow
```
[the remark this answers is not in the document]
Comment (amessbee) [reply main/guides/orchestration/how-orch-works.md]: @Jovonni fixed that. Also added a section on ICQ. Let me know if all looks.

File: main/guides/orchestration/how-orch-works.md [hunk 2]
```diff
@@ -1,37 +1,133 @@
...
+- **Automation:** Using ICA, the controller chain can automate operations
+  on the host chain. For instance, Agoric can programmatically manage assets
+  on another chain, automating workflows that span multiple blockchains.
+
+### Interchain Queries (ICQ)
+
+Interchain Queries (ICQ) is another powerful feature that enables a contract on
+one blockchain to query the state of an account another blockchain in a
+trustless manner. For example, a DeFi application on Agoric could use ICQ to
+fetch balance of an account from other chains like Osmosis or Cosmos Hub
```
Comment (Jovonni) [inline main/guides/orchestration/how-orch-works.md:106]:
```suggestion
fetch the balance of an account from other chains like Osmosis or Cosmos Hub
```
?

File: main/guides/orchestration/how-orch-works.md [hunk 3]
```diff
@@ -1,37 +1,133 @@
...
+without the need for autorization, enabling more sophisticated cross-chain
+trading strategies and risk management. ICQ enhances the interoperability of
```
Comment (Jovonni) [inline main/guides/orchestration/how-orch-works.md:108]:
```suggestion
interactions. ICQ enhances the interoperability of
```
wdyt?

**Review by Jovonni — APPROVED** [review]
lgtm

**Comment by amessbee** [comment]
@Jovonni thanks for the valuable feedback. I took care of the comments - will merge after checks!

**Pull request merged**